libbeat/processors/add_process_metadata/add_process_metadata.go

```diff
@@ -35,6 +35,8 @@ import (
 const (
 	processorName   = "add_process_metadata"
 	cacheExpiration = time.Second * 30
+	hostPath        = "/"
+	cgroupPrefix    = "/kubepods"
```

Comment:
This needs to be configurable too. I'd prefer to have this be search list so that it works out of the box with multiple tools (`/kubepods`, `/docker`). I don't know how the other tools name their cgroups, but if we can cover them too this would be great.